app/scripts/controllers/app-state.js

```diff
@@ -134,6 +169,69 @@ export default class AppStateController extends EventEmitter {
     this._resetTimer();
   }
 
+  _checkShouldShowRecoveryPhraseReminder() {
+    const {
+      recoveryPhraseReminderHasBeenShown,
+      recoveryPhraseReminderLastShown,
+    } = this.store.getState();
+
+    // For the first interval run, set recoveryPhraseReminderLastShown to the current time.
+    if (recoveryPhraseReminderLastShown === 0) {
+      this.store.updateState({
+        recoveryPhraseReminderLastShown: new Date().getTime(),
+      });
+      return;
+    }
+
+    // Capture the current timestamp
+    const currentTime = new Date().getTime();
```

Comment:
Nit: we could move this var declaration above `if (recoveryPhraseReminderLastShown === 0) {` and then use it on line 181.

Reply:
good catch, i can update this

Reply:
Updated